core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -52,7 +53,7 @@ class GroupMetadataManager(brokerId: Int,
                            interBrokerProtocolVersion: ApiVersion,
                            config: OffsetConfig,
                            replicaManager: ReplicaManager,
-                           zkClient: KafkaZkClient,
+                           private[group] var zkClient: KafkaZkClient,
```

Comment:
Did you make this a var for the test case? Wouldn't it be easier to pass the mocked instance directly?